WinUIGallery/ControlPages/IconElementPage.xaml.cs

```diff
@@ -22,5 +22,17 @@ public IconElementPage()
         {
             this.InitializeComponent();
         }
+
+        private void MonochromeButton_Checked(object sender, RoutedEventArgs e)
+        {
+            SlicesIcon.ShowAsMonochrome = true;
```

Comment:
This can be simplified with `MonoChromeButton_CheckedChanged` and setting:
`SlicesIcon.ShowAsMonochrome = (bool)MonochromeButton.IsChecked`

Reply:
Very good idea, I've updated that now.